NEWS.md

```diff
@@ -1,11 +1,42 @@
 MXNet Change Log
 ================
+## 0.12.0
+### New Features - Sparse Tensor Support
+  - Added limited cpu support for two sparse formats for Symbol and NDArray - CSRNDArray and RowSparseNDArray
+  - Added a sparse dot product operator and many element-wise sparse operators
+  - Added a data iterator for sparse data input - LibSVMIter
+  - Added three optimizers for sparse gradient updates: Ftrl, SGD and Adam
+  - Added push and row_sparse_pull with RowSparseNDArray in distributed kvstore
+  - For more information see [full release notes](https://cwiki.apache.org/confluence/display/MXNET/MXNet+0.12.0+Release+Notes)
```

Comment:
+  - For more information see [full release notes](https://cwiki.apache.org/confluence/display/MXNET/MXNet+0.12.0+Release+Notes)
Why is this put under sparse? Should we move it to the end?